README.md

```diff
@@ -87,9 +88,15 @@ software for Mac
 * [gitpitch.com](https://gitpitch.com/): Create compelling slideshow presentations in no time using the tools you already know and love ~ Markdown + Git
 
 ## Real time interactions
-
 * [kahoot.com](https://kahoot.com/): Game-based platform that makes learning awesome for millions of people all over the world. **Useful for real time surveys**
 
+## CFPs
```

Comment:
I would change this section title to be `CFPs and Speaking Resources`

Reply:
I like it. Done. 👍